src/context/LocaleContext/useLocal.ts

```diff
@@ -0,0 +1,10 @@
+import { useContext } from 'react'
+import { LocaleContext } from './LocalContext'
+
+export const useLocal = () => {
```

Comment:
Shouldn't this, and the file, called `useLocale` (missing an "e" at the end)?

Also, I think this could probably be in the `LocaleContext.ts` file since it's so tightly connected to that and makes no sense on its own? 🤔

Reply:
Haha, that is a typo, thanks! 😂 🙌 
I'm just used to writing hooks in it's own file I think, but I'm good with either solution 👍